Tasks/TerraformTask/TerraformTaskV4/task.json

```
@@ -14,7 +14,7 @@
     "demands": [],
     "version": {
         "Major": "4",
-        "Minor": "228",
+        "Minor": "229",
```

The instructions I got from the team handing over  this to us was that we should use azure devops sprint number. (https://whatsprintis.it/)

```suggestion
        "Minor": "236",
```